### PR TITLE
Jeffgolden/ch35161/migrate graphql to vendor api

### PR DESCRIPTION
**Pull request by jeffreygolden**

### CODE REVIEW

**Comment by jeffreygolden** [comment]
Requires https://github.com/replicatedhq/vandoor/pull/1465 for tests to pass

File: pkg/types/installer.go [hunk 1]
```diff
@@ -13,3 +15,41 @@ type InstallerSpec struct {
...
+	Yaml string `json:"yaml"`
+}
+
+type PromoteInstallerRequest struct {
+	Sequence     int64  `json:"sequence"`
+	VersionLabel string `json:"versionLabel"`
+	ChannelID    string `json:"channelId"`
+}
+
+type EnterpriseChannel struct {
```
Comment (sgalsaleh) [inline pkg/types/installer.go:33]: why are these here? graphql requests don't interact with the enterprise api

Comment (jeffreygolden) [reply pkg/types/installer.go]: they were provisional during dev, forgot to remove them after I found they weren't needed.  Removed now.
